hyperjumptech/monika
# Split the Node.js workflow jobs

# Monika Pull Request (PR)

## What feature/issue does this PR add
1. This PR splits the jobs in Node.js CI workflow. This way it's easier to read the logs for each of the jobs and the workflow is also slightly faster.

## How did you implement / how did you fix it
1. Split the jobs to lint, format, test, prod_test_npm, and prod_test_binary.

## How to test
1. Check the result in [https://github.com/hyperjumptech/monika/actions/runs/6782150319](https://github.com/hyperjumptech/monika/actions/runs/6782150319). You can see that the workflow are now split into 5 jobs that run in parallel.
